src/porepy/__init__.py

```diff
@@ -160,6 +160,7 @@
 from porepy.numerics.ad.equation_system import EquationSystem
 from porepy.numerics.ad._ad_utils import set_solution_values
```

Comment:
Could we merge these three lines to one?

Reply:
Can be merged into 1 import statement, but still spanning multiple lines because of the character limit.

Reply:
Merged into 1 import statement (for module `_ad_utils`) after discussion